### DISCUSSION TITLE
Pipe the incoming audio from iTunes to raspberry pi Bluetooth connected Alexa

### DISCUSSION THREAD

**Discussion by arnieSkyNet** [mikebrady/shairport-sync#1412] [Ideas]
I have my raspberry pi 3b connected to my Alexa echo dot and if I play any audio via vlc or aplay the sound comes out of my Alexa echo dot.

I made shairport with std-out and pipe

My iTunes sees the pi as speakers

What I hope to do is change a setting on the pi shairport to make the audio output to the Bluetooth connection on the alsa, any ideas on how?

![2C7B3C10-B117-4E18-A6F6-BE2187B9F7EA](https://user-images.githubusercontent.com/54072027/150612124-ae83be85-fea9-4384-a034-ed8d108777a0.jpeg)

**Comment by arnieSkyNet** [comment]
Here is a screen shot of my settings.

aplay does play audio to the selected Alaxa and Alexa appears as speakers on my win10 iTunes 

any help would be helpful :)
![61CD2134-FB03-4E41-BA57-B90C74827CB6](https://user-images.githubusercontent.com/54072027/151632837-8f0cdcc4-28e7-415d-9404-c8849af8a20b.jpeg)

**Reply by charlesomer** [reply]
When I tried to do this, I had to set up a loop back audio device which Shairport Sync can play to. Then a service which monitored the connecting loop back device and played out to the Bluetooth speaker (via aplay I think) 

Although I was also running Shairport Sync in a docker container
